docs/README.md

```diff
@@ -0,0 +1,9 @@
+# Quickstart
+
+* [Use Cluster API vSphere Provider using self-service workflow](self-service/README.md)
+  - [asciinema demo](https://asciinema.org/a/215805)
+  
+* [Use Cluster API vSphere Provider using management cluster workflow](ova/README.md)
+  - Cluster API vSphere Provider: Part 1 General Discussion (coming)
+  - [Cluster API vSphere Provider: Part 2 OVA ](https://www.youtube.com/watch?v=qn9A1bmYDe8)
+  - [Cluster API vSphere Provider: Part 3 Deploy a software defined cluster](https://www.youtube.com/watch?v=0ns35uEOn-A)
```

Comment:
i guess `software defined cluster` is a new way of saying target cluster?  any thought on switching?
other than this, /lgtm

Reply:
yeah, maybe I can use the same common terminology we've been using up till now.